pkg/controller/v1beta1/trainedmodel/shard/shard_manager.go

```diff
@@ -0,0 +1,21 @@
+package shard
+
+import (
+	v1beta1api "github.com/kubeflow/kfserving/pkg/apis/serving/v1beta1"
+)
+
+type ShardingStrategy string
+
+const (
+	Memory ShardingStrategy = "Memory"
+)
+
+type ShardManager struct {
```

Comment:
Consider calling this class ShardingStrategy

Reply:
I renamed the package to sharding and renamed this struct to Strategy so I can use it as sharding.Strategy